src/test/java/com/wikia/webdriver/testcases/discussions/mobile/PostDetailAnon.java

```diff
@@ -0,0 +1,8 @@
+package com.wikia.webdriver.testcases.discussions.mobile;
+
+/**
+ * Created by qaga on 2015-11-24.
+ */
+public class PostDetailAnon {
```

Comment:
Why you need empty class? Best practice is to not create placeholders like this. Please remove this and create when you will need it.